src/components/CampusAudioTour.tsx

```
@@ -0,0 +1,201 @@
+        fadeOutActiveAudio();
+      }
+    } catch (err) {
+      console.error('Proximity evaluation failure:', err);
+    }
+  };
+
+  // 3. Web Audio API Core Engine: Load and smoothly fade in media files
+  const triggerSpatialAudioStory = async (node: Milestone) => {
+    try {
+      fadeOutActiveAudio(); // Clear existing loops safely
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)